docs/reference/builder.md

```diff
@@ -1375,7 +1374,7 @@ useful interactions between `ARG` and `ENV` instructions:
 ```
 
 Unlike an `ARG` instruction, `ENV` values are always persisted in the built
-image. Consider a docker build without the --build-arg flag:
+image. Consider a docker builds without the --build-arg flag:
```

Comment:
This is not correct "a docker build" is singular.

Please change this back to what it was before